### ISSUE TITLE
"Hangouts Chat" Groups not visible in Pidgin

### ISSUE THREAD

**Issue by EionRobb** [EionRobb/purple-hangouts-attempt2#215]
**[Original report](https://bitbucket.org/EionRobb/purple-hangouts/issue/215) by aqeeliz (Bitbucket: [aqeeliz](https://bitbucket.org/aqeeliz), GitHub: [aqeeliz](https://github.com/aqeeliz)).**

----------------------------------------

Not sure if this is in the scope of plugin or not, but groups from “Hangouts Chat”, the GDrive / slack-like version of Hangouts, don’t show in Pidgin.

They don’t show in “Room List” when trying “Join a Chat”, or when already joined from web, no chat window opens when a message is received \(or sent\) in them.

Basically, they simply appear to be not implemented in the plugin.
